cmd/zetaclientd/utils.go

```diff
@@ -199,5 +202,40 @@ func CreateChainObserverMap(
 		}
 	}
 
+	// FIXME_SOLANA: config chain params
+	solChain, solConfig, enabled := appContext.GetSolanaChainAndConfig()
+	solChainParams := observertypes.ChainParams{
+		GatewayAddress: "94U5AHQMKkV5txNJ17QPXWoh474PheGou6cNP2FEuL1d",
+		IsSupported:    true,
+		ChainId:        solChain.ChainId,
+		InboundTicker:  10,
+	}
+
+	// create Solana chain observer if enabled
+	if enabled {
+		rpcClient := solrpc.New(solConfig.Endpoint)
+		if rpcClient == nil {
+			// should never happen
+			return nil, fmt.Errorf("solana create Solana client error")
```

Comment:
Can we use logger and continue like it is the behavior for EVM and Bitcoin?

Reply:
same behavior for EVM and Bitcoin. I'll still just print a log and ignore Solana error before the chain activation mechanism takes over this.

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a